packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,310 @@ components:
+            custom name or title
+          example: filename.mp4
+        meta:
+          type: object
+          description: User input metadata associated with the asset
+          example:
+            {
+              "title": "My awesome video",
+              "description": "This is a video of my awesome life",
+              "tags": ["awesome", "life", "video"],
+            }
+        updatedAt:
+          type: number
+          description:
+            Timestamp (in milliseconds) at which the asset was last updated
+          example: 1587667174725
+        createdAt:
+          type: number
+          readOnly: true
+          description: Timestamp (in milliseconds) at which asset was created
+          example: 1587667174725
+        hash:
+          type: array
+          description: Hash of the asset
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              hash:
+                type: string
+                description: Hash of the asset
+                example: 9b560b28b85378a5004117539196ab24e21bbd75b0e9eb1a8bc7c5fd80dc5b57
+              algorithm:
+                type: string
+                description: Hash algorithm used to compute the hash
+                example: sha256
+        videoSpec:
+          type: object
+          additionalProperties: false
+          readOnly: true
+          description: Video metadata
+          properties:
+            format:
+              type: string
+              description: Format of the asset
+              example: mp4
+            duration:
+              type: number
+              description: Duration of the asset in seconds (float)
+              example: 23.8328
+            tracks:
+              type: array
+              description:
+                List of tracks associated with the asset when the format
+                contemplates them (e.g. mp4)
+              items:
+                type: object
+                additionalProperties: false
+                required:
+                  - type
+                properties:
+                  type:
+                    type: string
+                    description: type of track
+                    enum:
+                      - video
+                      - audio
+                    example: video
+                  width:
+                    type: number
+                    description: Width of the track - only for video tracks
+                    example: 1920
+                  height:
+                    type: number
+                    description: Height of the track - only for video tracks
+                    example: 1080
+                  fps:
+                    type: number
+                    description: Frame rate of the track - only for video tracks
+                    example: 30
+                  id:
+                    type: string
```

Comment:
What is this track ID? ffprobe apparently doesn't return anything like that.

Reply:
Yeah we don't need it, there's not reference to it elsewhere, removed!